pages/system/dropdowns.js

```diff
@@ -114,24 +129,36 @@ _handleChange = (e) => {
         <System.P>Declare the Dropdown component.</System.P>
         <br />
         <System.CodeBlock>
-          {`<System.SelectMenu
-  name="one"
-  value={this.state.one}
-  category="horoscope"
-  onChange={this._handleChange}
-  options={SELECT_MENU_OPTIONS}>
-  {SELECT_MENU_MAP[this.state.one]}
-</System.SelectMenu>
+          {`const Select = () => {
```

Comment:
I would prefer `Select` is changed to `ExampleOne`, the stateless function component is meant to be served as an example they can copy and paste.